buddypress/common/js-templates/messages/parts/bp-messages-form.php

```diff
@@ -53,13 +53,13 @@ class="send-to-input"
 	<div id="bp-message-content"></div>
 
 	<?php
-	if ( 'mac' === $os ) {
+	if ( 'window' === $os ) {
```

Comment:
I'm not familiar with the code, so I don't know, but is it supposed to be window, not windows?

Reply:
Thanks for checking this important level of detail! Surprisingly enough, 'window' is correct.

Reply:
Good to know!